frontend/src/components/TopicCard/Desc.js

```diff
@@ -26,7 +26,7 @@ class MdTextArea extends RIETextArea {
 class Desc extends Component {
   render = () => {
     const descHTML = (!this.props.desc && this.props.authorizedToEdit)
-      ? '<p>Click to add description...</p>'
+      ? '<p class="emptyDesc">Edit the description...</p>'
```

Comment:
did you feel like some of this was easier because it's in react? I'm interested to hear if you think the developer experience has gotten better for you for this kind of sweeping change

Reply:
yeah 👍  👍  for sure